packages/suite/src/views/wallet/coinmarket/savings/bank-account/index.tsx

```diff
@@ -0,0 +1,124 @@
+    return (
+        <>
+            {isWatchingKYCStatus && <KYCInProgress />}
+            <Header>
+                <Translation id="TR_SAVINGS_BANK_ACCOUNT_HEADER" />
+            </Header>
+            <form onSubmit={handleSubmit(onSubmit)}>
+                <Controller
+                    control={control}
+                    name={typeSelectName}
+                    defaultValue={buildUsBankAccountType(UsBankAccountTypes[0])}
+                    render={({ onChange, value }) => (
+                        <>
+                            <Select
+                                options={UsBankAccountTypes.map(buildUsBankAccountType)}
+                                value={value}
+                                isClearable={false}
+                                hideTextCursor
+                                label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_TYPE_LABEL" />}
+                                onChange={(selected: any) => {
+                                    onChange(selected);
+                                }}
+                            />
+                            <BankAccountTypeNote>
+                                {value.value === UsBankAccountTypes[1] && (
+                                    <Translation id="TR_SAVINGS_BANK_ACCOUNT_TYPE_NOTE" />
+                                )}
+                            </BankAccountTypeNote>
+                        </>
+                    )}
+                />
+                <Input
+                    placeholder="Checking account"
+                    label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_NAME_LABEL" />}
+                    name={nameInputName}
+                    innerRef={register()}
+                    inputState={getInputState(errors.name, name)}
+                    bottomText={<InputError error={errors[nameInputName]} />}
+                />
+                <Input
+                    label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_ROUTING_NUMBER_LABEL" />}
+                    name={routingNumberInputName}
+                    inputState={getInputState(errors.routingNumber, routingNumber)}
+                    innerRef={register({
+                        required: 'TR_SAVINGS_BANK_ACCOUNT_ROUTING_NUMBER_REQUIRED',
+                    })}
+                    bottomText={<InputError error={errors[routingNumberInputName]} />}
+                />
+                <Input
+                    label={<Translation id="TR_SAVINGS_BANK_ACCOUNT_ACCOUNT_NUMBER_LABEL" />}
+                    name={accountNumberInputName}
+                    inputState={getInputState(errors.accountNumber, accountNumber)}
+                    innerRef={register({
+                        required: 'TR_SAVINGS_BANK_ACCOUNT_ACCOUNT_NUMBER_REQUIRED',
+                    })}
+                    bottomText={<InputError error={errors[accountNumberInputName]} />}
+                />
+                <Input
```

Comment:
nit: separating big components in the JSX output with empty lines improves readability